go-controller/pkg/node/gateway_udn.go

```diff
@@ -180,3 +185,112 @@ func (udng *UserDefinedNetworkGateway) deleteUDNManagementPort() error {
 	klog.V(3).Infof("Removed management port mac address information of %s for network %s", interfaceName, udng.GetNetworkName())
 	return nil
 }
+
+// computeRoutesForUDN returns a list of routes programmed into a given UDN's VRF
+// when adding new routes please leave a sample comment on how that route looks like
+func (udng *UserDefinedNetworkGateway) computeRoutesForUDN(vrfTableId int, mpLink netlink.Link) ([]netlink.Route, error) {
+	nextHops, intfName, err := getGatewayNextHops()
+	if err != nil {
+		return nil, fmt.Errorf("unable to get the gateway next hops for node %s, err: %v", udng.node.Name, err)
+	}
+	link, err := util.GetNetLinkOps().LinkByName(intfName)
+	if err != nil {
+		return nil, fmt.Errorf("unable to get link for %s, error: %v", intfName, err)
+	}
+	networkMTU := udng.NetInfo.MTU()
+	if networkMTU == 0 {
+		networkMTU = config.Default.MTU
```

Comment:
no I think we spoke about this and decided that for UDN's we should honor the user passed config OR default config and not add RoutableMTU into the mix...
the way I see it RoutableMTU is the config for default networks like NADMTU is the config for UDN's...
unless @trozet you say otherwise?